### PR TITLE
EDUCATOR-1705 Fix when profile_image is not available.

### PR DESCRIPTION
**Pull request by attiyaIshaque**
### Description
This PR fixes that when profile image of instructor is not available it would n't raises any error

### CODE REVIEW

File: course_discovery/apps/publisher/templates/publisher/_personFieldLabel.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div class="instructor" id="instructor_{{ staff.id }}">
     <input type="hidden" id="staff_{{ staff.id }}" name="staff" value="{{ staff.id }}">
-    <div><img src="{% if staff.profile_image_url %}{{ staff.profile_image_url }}{% else %}{{ staff.profile_image.url }}{% endif %}"></div>
+    <div><img src="{% if staff.profile_image %}{{ staff.profile_image.url }}{% else %}{{ staff.profile_image_url }}{% endif %}"></div>
```
Comment (schenedx) [inline course_discovery/apps/publisher/templates/publisher/_personFieldLabel.html]: You don't need the `if...else` statement here anymore. Please just use `{{ staff.get_profile_image_url }}`

**Review by schenedx — CHANGES_REQUESTED** [review]
See my comment

**Comment by attiyaIshaque** [comment]
@schenedx i have addressed your requested change.

**Review by schenedx — APPROVED** [review]
👍

**Pull request merged**